src/components/country-data-card/country-data-card.js

```diff
@@ -34,6 +40,31 @@ const CountryDataCardContainer = (props) => {
     });
   }, [])
 
+  useEffect(() => {
+    loadModules(["esri/layers/FeatureLayer"]).then(([FeatureLayer]) => {
```

Comment:
Do you think it would make sense to use the same hook for `countriesDataLayer` and `countriesDataLayer`? I get that using two hooks for two layers looks more readable, but we're loading `loadModules(["esri/layers/FeatureLayer"])` twice, am I understand that correctly?

Reply:
I would not update it right now since one of the hooks is gonna be removed once we have the data on the centroids layer, and for what I've seen on the `network` tab of the dev tools is not impacting (seems that the modules are being cached).